hyou/view.py

```diff
@@ -53,9 +55,7 @@ def _ensure_cells_fetched(self):
         response = self._api.sheets.spreadsheets().values().get(
             spreadsheetId=self._worksheet._spreadsheet.key,
             range=py3.str_to_native_str(range_str, encoding='utf-8'),
-            majorDimension='ROWS',
-            valueRenderOption='FORMATTED_VALUE',
-            dateTimeRenderOption='FORMATTED_STRING').execute()
+            **self._fetch_params).execute()
```

Comment:
Before the changes of this PR the default value of `dateTimeRenderOption` was `FORMATTED_STRING`. After the changes the default behaviour is to not specify this argument, in which case the [API default value  `SERIAL_NUMBER`](https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets.values/get#query-parameters) is used.

As this is a change in the default behaviour I would say this is a breaking change, therefore I would say we either have to change it or deploy with a new major.

What do you think @ddoskind ?

Reply:
The documentation is actually somewhat misleading. `dateTimeRenderOption` is only defaulting to `SERIAL_NUMBER` if `valueRenderOption` is *not* set to its default value `FORMATTED_VALUE`.

If `valueRenderOption` *is* `FORMATTED_VALUE`, then you always get formatted dates, no matter what you set `dateTimeRenderOption` to.

So I'd say it's not a change in default behavior.

Reply:
Basically `dateTimeRenderOption` is only for the scenario where you want unformatted values / formulas *except* for date values.

Reply:
You are right 😄